src/test/java/ca/bc/gov/backendstartapi/endpoint/CheckEndpointIT.java

```diff
@@ -17,21 +17,21 @@
 @WebFluxTest(CheckEndpoint.class)
 class CheckEndpointIT {
```

Comment:
The suffix *IT* doesn't follow the camel case rules from the style guide (see the third step of https://google.github.io/styleguide/javaguide.html#s5.3-camel-case).

On another note, the guide says that test classes end with the *Test* suffix, although the style checker didn't raise any issues in this point.

Reply:
Nice. Fixed, thanks!